translations/en/module_en_lang.php

```diff
@@ -93,6 +93,8 @@
     'wd_first-name' => 'First Name',
     'wd_gender' => 'Gender',
     'wd_heading_title' => 'Wirecard',
+    'wd_heading_title_payolution_b2b' => 'Wirecard Guaranteed Invoice (Payolution B2B)',
+    'wd_heading_title_payolution_invoice' => 'Wirecard Guaranteed Invoice  (Payolution B2C)',
```

Comment:
Double space in this translation looks like a "typo" (I guess it's already in PhraseApp like that?)

Reply:
changed it on PhraseApp